### PR TITLE
Adding new Beta environment

### PR DESCRIPTION
**Pull request by stjohnjohnson**
This adds an API and UI for beta

### CODE REVIEW

**Comment by jer** [comment]
👍

File: api-beta.yaml [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+    metadata:
+      labels:
+        app: screwdriver
+        tier: api
+        env: beta
+    spec:
+      serviceAccount: sd-build
+      containers:
+      - name: screwdriver-api
+        image: screwdrivercd/screwdriver:v3.1.8
```
Comment (d2lam) [inline api-beta.yaml]: We are pinning this?

Comment (stjohnjohnson) [inline api-beta.yaml]: It needs to be something, it is auto updated in the pipeline

Comment (d2lam) [inline api-beta.yaml]: True

**Pull request merged**